homeassistant/components/frontend/www_static/translations/en.json

```diff
@@ -0,0 +1 @@
+{"panel.states":"States","panel.map":"Map","panel.logbook":"Logbook","panel.history":"History","panel.log_out":"Log Out"}
```

Comment:
Translations for the frontend should be kept in the polymer repo. I want to move the whole `www_static` folder there, including the `script/build_frontend` script.

Reply:
This is just the generated output of `script/build_frontend`. The main source is in the polymer repo.